spec/spec_helper.rb

```diff
@@ -1,12 +1,15 @@
 ENV["RAILS_ENV"] ||= "test"
 ENV["GOVUK_APP_DOMAIN"] = "test.gov.uk"
 
+# Must go at top of file
+require "simplecov"
+SimpleCov.start "rails"
```

Comment:
Ooh what does the "rails" do?

Reply:
It appears to set [a custom profile](https://github.com/simplecov-ruby/simplecov#profiles) that does a bit of filtering. Moreover, it also seems to modify the way it collects coverage - I got very different results without it in Smart Answers!